docs/enhancements/ai-on-zero-worker-hypershift.md

```diff
@@ -0,0 +1,118 @@
+---
+title: ai-on-zero-worker-hypershift
+authors:
+  - "@slaviered"
+creation-date: 2022-08-28
+last-updated: yyyy-mm-dd
+---
+#  Installing the Infrastructure Operator on zero-node HyperShift (DRAFT)
+
+# Summary
+To support large numbers of HyperShift hosted control planes on a single management cluster, an architecture of nested control planes has been proposed.  The management cluster, which we refer to as L0, runs hypershift hosted control planes (L1). Spoke clusters are added to L1 control planes. The challenge is that we need to run the assisted-installer in L1 for adding workers to spoke clusters, but L1 doesn't have any nodes and therefore cannot run the installer directly.
+
+The architecture for multiple hubs is described in the following diagram
+<img src="https://github.com/xiangjingli/hyper-acm-deploy/blob/main/docs/images/acm-hosted-cluster-diagram.svg" width="30%" height="30%"/>
+
+The hosted control plane (L1) is responsible for managing the CRs and CRDs, where as the actual pods are located in dedicated namespaces and managed by the hosting management cluster (L0).
+
+The deployment and cluster provisioning on such a setup is described in this [POC](https://github.com/javipolo/openshift-assisted-installer-tests/tree/main/ai-in-zero-worker-hypershift). For convinience, the architecure of the deployment is also pasted ![here:](../ai-on-hypershift.png).
+
+# Motivation
+## Goals
+- Automate the deployment of assisted service on zero-nodeworker hypershift clusters using the assisted infrastructure operator
+- Support Infrastructure Operatorai installation on multiple namespaces.
```

Comment:
```suggestion
- Support Infrastructure Operator installation in multiple namespaces.
```

Reply:
Also the goal isn't to install the infrastructure operator into multiple namespaces, right? It's to allow it to install multiple instances of the service. I think that might be what you're trying to say here, but I would make it more clear.

Reply:
Both fixed.